dbt/adapters/firebolt/__version__.py

```diff
@@ -1 +1 @@
-version = "0.21.3"
+version = '0.21.3'
```

Comment:
I really hope this is the last time we have to change quotes from double to single

Reply:
My comments on this have gone missing. Recreated now, sigh.

As context, PEP8 [states that it has no preference](https://pep8.org/#string-quotes), but plainly biases toward single quotes in its documentation. Also, here's an extensive discussion on Black's [git](https://github.com/psf/black/issues/118)

I personally prefer single quotes because of my history with coding and because I find double quotes to be visual clutter. That said, it seems that the convention _might_ be single quotes for data and double quotes for human-read strings, e.g. `some_dict['key']` vs output = "Some string." See [this comment](https://github.com/psf/black/issues/118#issuecomment-382349179) on the above referenced Black git.

I'm willing to bend to convention, as long as it's consistent.